snapshots/output/syntax/src/conflicting-const-interface.ts

```diff
@@ -0,0 +1,9 @@
+  export const ConflictingConst = 42
+// definition syntax 1.0.0 src/`conflicting-const-interface.ts`/
+//documentation ```ts\nmodule "conflicting-const-interface.ts"\n```
+//             ^^^^^^^^^^^^^^^^ definition syntax 1.0.0 src/`conflicting-const-interface.ts`/ConflictingConst.
+//             documentation ```ts\ninterface ConflictingConst\n```
+  export interface ConflictingConst {}
+//                 ^^^^^^^^^^^^^^^^ definition syntax 1.0.0 src/`conflicting-const-interface.ts`/ConflictingConst.
```

Comment:
The expected behavior
```suggestion
//                 ^^^^^^^^^^^^^^^^ definition syntax 1.0.0 src/`conflicting-const-interface.ts`/ConflictingConst#
```